packages/modules-sdk/src/loaders/utils/load-internal.ts

```diff
@@ -0,0 +1,119 @@
+import { asFunction, asValue } from "awilix"
+import { createMedusaContainer } from "medusa-core-utils"
+import { trackInstallation } from "medusa-telemetry"
+import {
+  Constructor,
+  InternalModuleDeclaration,
+  Logger,
+  MedusaContainer,
+  ModuleExports,
+  ModuleResolution,
+  MODULE_RESOURCE_TYPE,
+  MODULE_SCOPE,
+} from "../../types"
+
+export async function loadInternalModule(
+  container: MedusaContainer,
+  resolution: ModuleResolution,
+  logger: Logger
+): Promise<{ error?: Error } | void> {
+  const constainerName = resolution.definition.registrationName
```

Comment:
todo: this is not the name of the container but the name of the registration, should be registrationName or resolutionKey

Reply:
it is the case for all containerName in the different files

Reply:
I'm not sure if the options to change the registration name is actually good. This can cause a lot of confusion an bugs when working with the modules

Reply:
no the registrationName is well named, just the const that is not

Reply:
Yeah, I got it, but I mean overall it is confusing to have a different name to register the container other than the service name, as other modules might expect that to be injected. We could have aliases, but I don't see much value on keeping this variable.

Reply:
the registrationName is not the name of the container. The container does not have a name, it is the name of the registration (service for example). So I don't understand what you are saying about a different name? I just mean to rename the const name :)

Reply:
Did I miss understood your answer?

Reply:
Module definition has key and registrationName:

```
{
    key: "inventoryService",
    registrationName: "inventoryService",
    defaultPackage: false,
    label: "InventoryService",
    isRequired: false,
    canOverride: true,
    dependencies: ["eventBusService"],
    defaultModuleDeclaration: {
      scope: MODULE_SCOPE.INTERNAL,
      resources: MODULE_RESOURCE_TYPE.SHARED,
    },
```
this can be replaced on medusa-config, I'm saying we should use the "key" for everything and not let registrationName change

Reply:
yes but this is not the subject of my comment, I just asked to rename the `constant` named `containerName` since it is not the name of the container. So i don't understand the link between the two? Do I miss something?

Reply:
hahaha sorry... I was just thinking out loud.
Will address this in other PR and check with @olivermrbl if we can remove `registrationName`

Reply:
Yeah the key can be the name it will be registered on and be non changeable, and registrationName could be removed. since they are the same all the time